umm-lib/resources/schema/echo10/Granule.xsd

```diff
@@ -353,27 +353,29 @@ xmlns:xs="http://www.w3.org/2001/XMLSchema">
       characteristics associated with a granule.</xs:documentation>
     </xs:annotation>
     <xs:sequence>
-      <xs:element minOccurs="0" name="DataGranuleSizeInBytes"
-      type="xs:unsignedLong">
+      <xs:element minOccurs="0" name="DataGranuleSizeInBytes" type="xs:unsignedLong">
         <xs:annotation>
-          <xs:documentation>The attribute will indicate the
-          volume of data in bytes contained in the
-          granule.</xs:documentation>
+          <xs:documentation>The size in Bytes of the volume of data contained in the granule.
+          Bytes are defined as eight bits. Please use this element instead of or inclusive 
```

Comment:
I don't know if it matters, but is a signed byte important here?

Reply:
Granule.xsd file is strictly from what Erich merged. I have no control over it.  If you didn't like it, you should have made the comment in his PR :)

Reply:
I will write a bug ticket then and have him change the schema on his side.